src/parser/csql_grammar.y

```diff
@@ -18455,7 +18424,12 @@ predicate_expr_sub
 				      }
 				    else
 				      {
-					found_paren_set_expr = true;
+					if (subq)
+                                          {
+                                            /* If not PT_TYPE_STAR */
+                                            pt_select_list_to_one_col (this_parser, opd1, true);     
+                                          }                                        
+                                        found_paren_set_expr = true;
```

Comment:
It looks indentation broken because of the use of tab.

Reply:
i fixed it